artemis-core-client/src/main/java/org/apache/activemq/artemis/core/server/management/NotificationService.java

```diff
@@ -37,4 +37,8 @@
 
    void removeNotificationListener(NotificationListener listener);
 
+   default void callNotificationListener(Notification notification) {
```

Comment:
This is purpose of sendNotification, any new thing should integrate with the existing way things work.